ext/web/06_streams.js

```diff
@@ -4863,6 +4951,43 @@ class ReadableStream {
     }
   }
 
+  static from(asyncIterable) {
+    webidl.requiredArguments(
+      arguments.length,
+      1,
+      "Failed to call 'ReadableStream.from'",
+    );
+    asyncIterable = webidl.converters.any(asyncIterable);
+
+    const iterator = getIterator(asyncIterable, true);
+
+    const stream = createReadableStream(() => undefined, async () => {
+      // deno-lint-ignore prefer-primordials
+      const res = await iterator.next();
+      if (typeof res !== "object") {
+        throw new TypeError("iterator.next value is not an object");
+      }
+      if (res.done) {
+        readableStreamDefaultControllerClose(stream[_controller]);
+      } else {
+        readableStreamDefaultControllerEnqueue(stream[_controller], res.value);
+      }
+    }, async (reason) => {
+      if (typeof iterator.return === "undefined") {
+        return undefined;
+      } else {
+        // deno-lint-ignore prefer-primordials
+        const res = await iterator.return(reason);
+        if (typeof res !== "object") {
+          throw new TypeError("iterator.return value is not an object");
+        } else {
+          return undefined;
+        }
+      }
+    });
```

Comment:
```suggestion
    }, 0);
```

`highWaterMark` defaults to 1 but should be 0.